### PR TITLE
Board list

### PR DESCRIPTION
**Pull request by AndriyKap**

### CODE REVIEW

**Review by ishandry — APPROVED** [review]

**Review by OlenaHatala — APPROVED** [review]

File: front-end/frontend/src/modules/boards/components/table_info/TableInfo.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  if(props.counters?.tickets)
+  {
+    tickets = <IconCounter icon={priceTag} counter={props.counters.tickets}/>;
+  }
+
+  return (
+    <div className={classes.table}>
+      <div className={classes["table-details"]}>
+      <span className={classes["calendar-icon"]}><Icon icon={calendar} size={50} style={{ color: '#1C1333' }}/></span>
```
Comment (daftkid) [inline front-end/frontend/src/modules/boards/components/table_info/TableInfo.js:26]: This value of the color is repeatedly in use within different files. Does it make sense to have it defined in some global CSS file?

File: front-end/frontend/src/modules/boards/components/table_info/TableInfo.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                }
+                </div>
+            ) : null}
+
+          </div>
+          <div className={classes["service-info"]}>{props.servName}</div>
+          <div className={classes["address-info"]}>{props.address}</div>
+          
+        </div>
+        <span className={classes["circleRight-icon"]}><Icon icon={circleRight} size={36} style={{ color: '#717171' }}/></span>
```
Comment (daftkid) [inline front-end/frontend/src/modules/boards/components/table_info/TableInfo.js:49]: The same situation here

**Review by daftkid — CHANGES_REQUESTED** [review]

File: front-end/frontend/src/modules/boards/components/board_list/BoardList.module.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  border: 1px solid #f3f3f3;
+}
+
+.noactive-button:hover {
+  background-color: #1f1638;
+}
+
+
+
+
```
Comment (daftkid) [inline front-end/frontend/src/modules/boards/components/board_list/BoardList.module.css:40]: I neeeeeed more space!!!

**Review by daftkid — CHANGES_REQUESTED** [review]

File: front-end/frontend/src/modules/common/hooks/index.js
Comment (daftkid) [inline front-end/frontend/src/modules/common/hooks/index.js:1]: why is this empty?

**Review by daftkid — CHANGES_REQUESTED** [review]

File: front-end/frontend/src/modules/common/hooks/index.js
Comment (ishandry) [reply front-end/frontend/src/modules/common/hooks/index.js:1]: fixed

File: front-end/frontend/src/modules/home/pages/HomeIndexPage/HomeIndexPage.module.css
Comment (daftkid) [inline front-end/frontend/src/modules/home/pages/HomeIndexPage/HomeIndexPage.module.css:1]: This one is still empty :)

**Review by daftkid — CHANGES_REQUESTED** [review]

**Pull request merged**